_data/simple-icons.json

```
@@ -8168,6 +8168,11 @@
             "hex": "6CAC4D",
             "source": "https://github.com/keepassxreboot/keepassxc/tree/3fdafc6d25e85050976e0cc645db579086db3f45"
         },
+        {
+            "title": "KeeWeb",
+            "hex": "528BFF",
+            "source": "https://github.com/keeweb/keeweb/blob/44432eb66d5b771d5867cdd74f2500d00e006783/app/icons/safari-pinned-tab.svg"
```

Seems like a better file to grab as a source, looks identical to the pinned tab one anyway. Wish it was in color!
```suggestion
            "source": "https://github.com/keeweb/keeweb/blob/44432eb66d5b771d5867cdd74f2500d00e006783/graphics/svg/keeweb.svg"
```